server/game/cards/06-WoE/RecordedHistory.js

```diff
@@ -0,0 +1,49 @@
+const Card = require('../../Card.js');
+
+class RecordedHistory extends Card {
+    // Play: Reveal up to 3 cards of different houses from your
+    // hand. Archive each card revealed this way.
+    setupCardAbilities(ability) {
```

Comment:
I believe there is a better way to implement this. I will check. The problem with this implementation is, besides verbose, it will archive 1 at a time and who knows in the future there could be an archive interrupt.

Reply:
Thanks, would love to make this cleaner.

Reply:
Take a look at https://github.com/keyteki/keyteki/pull/3211

Reply:
Ok thanks, I changed it to use the `BaseCardSelector`.  The only drawback is that the card still looks "selectable" according to the prompts -- it just doesn't do anything when you click it.  Hopefully that's ok. (I assume Hallafest will work the same way.)